README.md

```diff
@@ -71,7 +71,7 @@ there, please contact support for Windows driver information.
 
 - [Intel Chipset Driver](./drivers/chipset-10.1.18950.8298-public-mup.zip)
   - Download the ZIP file, extract, and run `SetupChipset.exe`
-- [Intel SerialIO Driver](./drivers/serialio_30.100.2031.2_pv_tgl-lp_20h1v2.zip)
+- [Intel SerialIO Driver](./drivers/serialio_30.100.2031.2_pv_adl-lp_22h2.zip)
```

Comment:
All other boards using this file would require this change.

Reply:
Should be fixed in the latest commit.

Reply:
So is there any reason for this rename then? It's fine to have the ADL boards use files with TGL in the name.

(Or even, why do we include the chipset ID in the file name?)